client/src/components/Visualization/charts/Geochart.js

```diff
@@ -48,9 +51,17 @@ const Geochart = () => {
 
       for (let [stateTotal, topAdvertiser] of zippedResults) {
         if (stateTotal.data() !== undefined) {  
-          const tooltip = `Total Ad Spend (USD): ${stateTotal.data().totalStateSpend}
-              Top Advertiser: ${topAdvertiser.docs[0].id}`;
-          dataTable.push([stateTotal.id.toUpperCase(), stateTotal.data().totalStateSpend, tooltip]); 
+
+          // Format total state ad spend as currency string.
+          const stateSpend = stateTotal.data().totalStateSpend;
```

Comment:
create a separate function for line 56 - 59; something like `formatStateSpend` then move all the code between line 56 - 59 into that function